test/SeqCli.Tests/Cli/Commands/VersionCommandTests.cs

```diff
@@ -1,13 +1,11 @@
-﻿#if !DEBUG
-
-using SeqCli.Cli.Commands;
+﻿using SeqCli.Cli.Commands;
 using Xunit;
 
 namespace SeqCli.Tests.Cli.Commands
 {
     public class VersionCommandTests
     {
-        [Fact]
+        [Fact(Skip = "dotnet test rebuilds binaries and this no longer works")]
```

Comment:
Can we just delete it?

Reply:
Just realised that we can probably move it to an end-to-end test; I'll try creating one, and delete this either way :+1: